examples/notebook/src/commands.ts

```diff
@@ -164,11 +166,19 @@ export const SetupCommands = (
   });
   commands.addCommand(cmdIds.extendAbove, {
     label: 'Extend Above',
-    execute: () => NotebookActions.extendSelectionAbove(nbWidget.content)
+    execute: () => NotebookActions.extendSelectionAbove(nbWidget.content, false)
+  });
+  commands.addCommand(cmdIds.extendTop, {
+    label: 'Extend Above',
```

Comment:
Can you change the label to Extend to Top, and similarly for the other new command?

Reply:
Sure.